DuckDuckGo/NetworkProtection/NetworkExtensionTargets/NetworkExtensionTargets/MacPacketTunnelProvider.swift

```diff
@@ -436,16 +417,60 @@ final class MacPacketTunnelProvider: PacketTunnelProvider {
         case .staging:
             subscriptionEnvironment.serviceEnvironment = .staging
         }
+        subscriptionEnvironment.purchasePlatform = .stripe // we don't care about the purchasePlatform
```

Comment:
I've no idea why we don't care about the purchase platform.  Could we document what this means, maybe providing a link with reference or a clearer explanation?

Reply:
Comment added: `The SysExt doesn't care about the purchase platform because the only operations executed here are about the Auth token. No purchase or platforms-related operations are performed.`